packages/shared/forks/ReactFeatureFlags.native-fb.js

```diff
@@ -48,7 +48,7 @@ export const disableNativeComponentFrames = false;
 export const skipUnmountedBoundaries = false;
 export const deletedTreeCleanUpLevel = 1;
 export const enableSuspenseLayoutEffectSemantics = false;
-
+export const enableGetInspectorDataForInstanceInProduction = true;
```

Comment:
This new export will only be enabled for Facebook builds. (It will be `undefined` for OSS builds.)